src/script/cpu-map.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
```

Comment:
Can you please move the new script into a separate commit?

Reply:
Sure, will do